README.md

```diff
@@ -178,6 +178,7 @@ public protocol FMAlertable {
 ```
 
 ## Apps using FMPhotoPicker
+If you using FMPhotoPicker in your apps, please tell we & please P-R add this list.
```

Comment:
I think something like this is good enough:
`If you are using FMPhotoPicker in your project, please let us know.`